docs/blockchain/contracts/nft-contract/nft-actions/mknftofr.a.experimental.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'mknftofr.a'
+order: 37
+
+---
+
+# mknftofr.a
+
+Make an offer on a given token.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the NFT factory from which the NFT was issued.
+
+The action stores the offer to `nftoffer.a` table with the specified arguments. The new offer ID is read from `next.nftofr` table whose value field is then incremented.
+
+`eosio.nftram` pays RAM usage.
+
+The offered price will be transferred to `eosio.nftofr` account and will be kept until the offer is either accepted or cancelled.
+
+An account will not be able to make offers on the same NFTs that they already made.
+
+An account will not be able to make more offers on NFTs than `max_active_offer_per_user` of `offercfg.a`, which records the global configurations.
+
+`owner` should be neither buyer nor receiver.
+
+`price` should be no less than min_price of `offercfg.a` and also should be no less than minimum_resell_price of the factory.
+
+`promoter_basis_point` should be in the range between `min_promoter_share_bp` and `max_promoter_share_bp` of `saleshrlmcfg` table configurations for resale. If `saleshrlmcfg` table doesn’t exist, the default range is between 250 (2.5 %) and 1000 (10 %).
+
+`duration` should be in the range between `min_duration` and `max_duration` of `offercfg.a`. If duration is longer than the trading window of the factory, it will be capped by `trading_window_end`.
+
+## Action Parameters
+
+| Property Name        | C++ Type        | JavaScript Type | Definition                                                                         |
+| -------------------- | --------------- | --------------- | ---------------------------------------------------------------------------------- |
+| buyer                | name            | String          | Account who make an offer                                                          |
```

Comment:
Account who make an offer 
should be
Account who makes an offer

Reply:
updated